client/package.json

```diff
@@ -14,7 +14,9 @@
     "build": "node build/build.js"
   },
   "dependencies": {
+    "axios": "^0.17.1",
     "express": "^4.16.2",
```

Comment:
Also, this one should not be necessary as well.

Reply:
ok